edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Integration.Test/ModulePriorityValidator.cs

```diff
@@ -11,11 +13,13 @@ public class ModulePriorityValidator : Validator
 
         public override bool Validate(TestCommandFactory testCommandFactory)
```

Comment:
Want to call out that this integration test was changed a lot. I swapped out the existing TestCommandFactory impl with the more robust one already existing in our other tests.